pkg/framework/simulator_test.go

```diff
@@ -24,14 +24,22 @@ import (
 	"k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 	fakeclientset "k8s.io/client-go/kubernetes/fake"
+	"k8s.io/component-base/version"
 	kubescheduleroptions "k8s.io/kubernetes/cmd/kube-scheduler/app/options"
 	apitesting "k8s.io/kubernetes/pkg/api/testing"
 	kubeschedulerconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"
-	"k8s.io/kubernetes/pkg/version"
 	"k8s.io/utils/pointer"
 )
 
+// func init() {
+// 	flag.Set("alsologtostderr", fmt.Sprintf("%t", true))
+// 	var logLevel string
+// 	flag.StringVar(&logLevel, "logLevel", "4", "test")
+// 	flag.Lookup("v").Value.Set(logLevel)
+// }
+
```

Comment:
do you want to remove this?

Reply:
I would rather keep it there so anyone can see klog in case it's needed.